### PR TITLE
Hello firstweek

### PR DESCRIPTION
**Pull request by richie1988**
Updated README.md file

### CODE REVIEW

**Comment by richie1988** [comment]
please go through my project and check what i need to submitt it looks like i have been doing some  changes over and over same thing please if there is any problem just tell me once cause i only used html and css on this project and i im not on social media or liknkedin account i have no account on any platform this is my first platform

File: README.md
Comment (tsohleDev) [inline README.md:1]: [Optional] I think you could be more descriptive in your PR title
e.g 
## Creating Mobile version

In this PR we:
- Make ...
- ...

[OPTIONAL] I also suggest you avoid creating new Pull requests when **changes are required** which means its also not advised to merge before you get approved

**Review by tsohleDev — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi Micronaut @richie1988 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

will happy

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — APPROVED** [review]
Hi @richie1988, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by richie1988** [comment]
Thanks

**Pull request merged**